src/output-json-sip.c

```diff
@@ -79,7 +79,8 @@ static int JsonSIPLogger(ThreadVars *tv, void *thread_data,
     LogSIPLogThread *thread = thread_data;
     json_t *js, *sipjs;
 
-    js = CreateJSONHeader(p, LOG_DIR_PACKET, "sip");
+    js = CreateJSONHeader(p, LOG_DIR_PACKET, "sip",
+                          thread->siplog_ctx->file_ctx->options_flags);
```

Comment:
Just below this will call JsonAddCommonOptions that a configuration parameter.  Can this be moved into that for now? Ideally this should be combined at some time.